.github/workflows/snapshot-release.yml

```diff
@@ -0,0 +1,39 @@
+name: Snapshot Release
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+on:
+  push:
+    branches: [ main, next ]
+
+jobs:
+  release-snapshot:
+    runs-on: ubuntu-latest
+    name: snapshot-release
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Import GPG key
+        id: import_gpg
+        uses: crazy-max/ghaction-import-gpg@v3
+        with:
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.GPG_PASSPHRASE }}
+
+      - uses: actions/setup-java@v2
+        with:
+          distribution: temurin
+          java-version: 11
+          cache: 'maven'
+
+      - name: Configure Git author
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
```

Comment:
I think you can drop this. You shouldn't need to commit anything with a simple deploy.

Otherwise it looks good.

Reply:
Thanks for the review!